region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -350,6 +363,19 @@
         }},
         "canShinechargeMovement"
       ],
+      "unlocksDoors": [
+        {"types": ["missiles"], "requires": ["never"]},
+        {"types": ["super"], "requires": []},
+        {
+          "types": ["powerbomb"],
+          "requires": [
+            {"or": [
+              "canCrumbleJump",
```

Comment:
Little bit harder than a normal crumblejump.  A walljump instant morph works too, I think.

Reply:
Makes sense. Added a walljump instant morph option, with a "canTrickyJump" requirement on it and on the crumblejump option.